src/http.py

```diff
@@ -1,34 +1,58 @@
 ###Finding records in Sierra that do not have working links###
 
+#10/11: formatted imports to match discussion
 import csv
+#10/11: had to pip install requests in this venv. Resolved
 import requests
 from pymarc import Field,Subfield,MARCReader
 
 class Rowish:
     def __init__(self,control,url,bib_code,suppressed,live):
         self.control = control,
-        self.url = url
+        #10/11: added a comma below. Oops
+        self.url = url,
         self.bib_code = bib_code,
         self.suppressed = suppressed,
         self.live = live
 
+###10/11: may be more helpful to use namedtuples
+#ex: 
+#Rowish_named = namedtuple("Rowish_named","control,url,bib_code,suppressed,live")
+#for row in map(Rowish_named._make,csv.reader(open("foo.csv","rb"))):
+#   print(row.control,row.url)
+
 def csv_classer(file):
     classed = []
     for row in file:
         new_row = Rowish(row[0],row[1],row[2],False,True)
         classed.append(new_row)
+    #Rowish_named = namedtuple("Rowish_named","control,url,bib_code,suppressed,live")
+    #for row in map(Rowish_named._make,csv.reader(open("file","rb"))):
+    #   classed.append(row)
     return classed
 
 def mrc_classer(file):
     classed = []
     for record in file:
         control = record.get_fields["001"]
```

Comment:
`control = record.get_fields["001"]` should be
`control = record.get_fields("001")`
You are using the .`get_fields()` method (parenthesis).
Square brackets are used when you access a dictionary by a key:
`record["001"]`

Reply:
Understood! Thank you.